unittests/test_policies.py

```diff
@@ -257,6 +260,38 @@ def test_dependencies(self):
             if t.ref_count == 0:
                 assert os.path.exists(os.path.join(check.outputdir, 'out.txt'))
 
+    def test_sigterm(self):
+        # Wrapper of self.runall which is used from a child process and
+        # passes any exception to the parent process
+        def _runall(checks, conn):
+            exc = None
+            try:
+                self.runall(checks)
+            except BaseException as e:
+                exc = e
+
+            conn.send(exc)
+            conn.close()
+
+        with pytest.raises(ReframeForceExitError,
+                           match='received TERM signal'):
+            rd_endpoint, wr_endpoint = multiprocessing.Pipe(False)
```

Comment:
Pass the `duplex` keyword so that the code is more self explanatory:

```suggestion
            rd_endpoint, wr_endpoint = multiprocessing.Pipe(duplex=False)
```